### ISSUE TITLE
Grunt 1.5.x doesn't satisfy what grunt-trimtrailingspaces require

### ISSUE THREAD

**Issue by waqas385** [paazmaya/grunt-trimtrailingspaces#72]
### Expected behaviour

Package should support latest grunt version 1.5.x

### Actual behaviour

grunt version 1.5.3, which doesn't satisfy what grunt-trimtrailingspaces requests

### Versions and environment

 * Operating system: MacOs
 * Node.js: 16.14.2
 * npm: 8.5.0
 * Grunt.js: 
 * grunt-trimtrailingspaces: latest

Thank you and have some :pizza:.

**Comment by waqas385** [comment]
Hi,

Any updates regarding upgrading grunt version 🥇

**Comment by paazmaya** [comment]
Thank you for the issue, but can you please elaborate more specifically what does this mean?

**Comment by waqas385** [comment]
I faces following error after upgrading node, grunt & npm version for my app:

`
....
npm ERR! Found: grunt@1.5.3
npm ERR! node_modules/grunt
npm ERR!   dev grunt@"^1.4.1" from the root project
npm ERR!
npm ERR! Could not resolve dependency:
npm ERR! peer grunt@"~1.1" from grunt-trimtrailingspaces@5.0.0
npm ERR! node_modules/grunt-trimtrailingspaces
npm ERR!   dev grunt-trimtrailingspaces@"^5.0.0" from the root project
npm ERR!
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
....
`

After disabling plugin from **gruntfile.js** it works fine. I didn't dig further because as per my understanding grunt-trimtrailspacing supports grunt 1.1 version and I have installed grunt 1.5.3 for app.
